src/test/java/org/george/jylishop/test/controllers/AdminHemoControllerTest.java

```diff
@@ -48,15 +43,10 @@ public void postFormTest() {
         String description = "Description";
         int id = 444;
         String picture = "Picture";
-
-        base.getCatalogue().clear();
-        assertEquals(base.getCatalogue().size(), 0);
-
-        when(utils.getProductById(id)).thenReturn(null);
+        int a=base.getCatalogue().size();
```

Comment:
Please, rename variable to smth more senseful than simple 'a'. And do not forget to format the code :)